src/site/content/en/blog/squoosh-v2/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Announcing Squoosh v2
+subhead: New codecs support, updatd design, and CLI support!
+description: |
+  Major version change to Squoosh.app with new CLI feature.
+scheduled: true
+date: 2020-12-09
+hero: hero.jpg
+alt: A screeen capture of new Squoosh top page
+tags:
+  - blog
+  - tools
+  - javascript
+  - images
+---
+
+
+
+[Squoosh](https://squoosh.app) is an image compression app our team built and [debuted at Chrome Dev Summit 2018](https://youtu.be/ipNW6lJHVEs). We built it to make it easy to experiment with different image codecs, and to showcase the capabilities of the modern web.  
+
+Today, we are releasing a major update to the app with more codecs support, a new design, and a new way to use Squoosh on your command line called Squoosh CLI.
+
+
+## New codecs support
+
+We now support OxiPNG, MozJPEG, WebP, and AVIF, in addition to codecs natively supported in your Browser.  A new codec was made possible again with the use of WebAssembly. By compiling a codec encoder and decoder as WebAssembly module users can access and experiment with newer codecs even if their preferred browser does not support them. 
+
+To learn more about different codecs, we are kicking off Understanding Image Codecs series on web.dev so stay tuned!
+
+
+## Launching a command line Squoosh!
+
+Ever since the original launch in 2018, common user request was to interact with Squoosh programmatically without UI. We felt a bit conflicted about this path since our app was a UI on top of command line based codec tools. However we do understand the desire to interact with the whole package of codecs instead of multiple tools. Squoosh CLI does just that.
```

Comment:
```suggestion
Ever since the original launch in 2018, a common user request was to interact with Squoosh programmatically without a GUI. We felt a bit conflicted about this path since our app was a UI on top of command line based codec tools. However we do understand the desire to interact with the whole package of codecs instead of multiple tools. Squoosh CLI does just that.
```